docs/src/references/mqtt-api.md

```diff
@@ -0,0 +1,838 @@
+  main["#8197;#8197;main#8197;#8197;"]
+
+  device --/--- main --/--- service --/--- service_id
+```
+
+  </td>
+  <td><code>te/device/main//</code></td>
+</tr>
+
+<!-- Main device service -->
+<tr>
+  <td>Service</td>
+  <td>
+
+```mermaid
+graph LR
+  main["#8197;#8197;main#8197;#8197;"]
+
+  device --/--- main --/--- service["service"] --/--- service_id["node_red"]
+```
+
+  </td>
+  <td><code>te/device/main/service/node_red</code></td>
+</tr>
+
+<tr><td colspan="4"></td></tr>
+<tr><td colspan="4"><strong>Child ("child01")</strong></td></tr>
+
+<!--  Child device -->
+<tr>
+  <td>Device</td>
+  <td>
+
+```mermaid
+graph LR
+  service["#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;"]
+  service_id["#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;#8197;"]
+
+  device --/--- child01 --/--- service --/--- service_id
+```
+
+  </td>
+  <td><code>te/device/child01//</code></td>
+</tr>
+
+<!-- Child device service -->
+<tr>
+  <td>Service</td>
+  <td>
+
+```mermaid
+graph LR
+  device --/--- child01 --/--- service["service"] --/--- service_id["node_red"]
+```
+
+  </td>
+  <td><code>te/device/child01/service/node_red</code></td>
+</tr>
+</table>
+
+
+### Channel {#group-channel}
+
+The channel group represents the information which is associated to the **identifier**.
+Information includes both telemetry data (e.g. measurement, events and alarms) 
+as well as commands which are used to execute actions on the device.
+
+The channel group is represented by the following segments:
+
+<p align="center">
+
+```mermaid
+graph LR
+  category["&lt;category&gt;"] --/--- type["&lt;type&gt;"] --/--- ...
+```
+
+</p>
+
+The individual **categories** dictate the subsequent topic structure and payload schema, however the MQTT topic schema strives to keep some consistency amongst the different categories by applying similar concepts where possible.
+
+|Segment|Description|
+|-------|-----------|
+|&lt;category&gt;|Category of data (telemetry or commands) which is related to the **identifier**. A fixed list of categories is used to represent data types such as; measurements, alarms, events, commands etc.|
+|&lt;type&gt;|A unique type/name used to identify the information being pushed or received. Types allow users to filter/subscribe to data which interests them. For example, a measurement could be published under a type called "flow_rate", then other clients interested in the flow rate can subscribe to that single typed topic.|
+|`...`|Additional channel specific topic segments. Each **category** is responsible for defining the number and meaning of the remaining topic segments.|
+
+
+#### Categories
+
+The following is an overview of the channel categories which are available.
+
+|Category|Description|
+|--------|-------|
+|m|Measurements|
+|e|Events|
+|a|Alarms|
+|health|Health/status|
+|cmd|Commands|
+|data|Process/inventory data|
```

Comment:
As mentioned in a previous comment, the term `data` is a bit too broad/vague to convey what it really is. Since we haven't even defined what all it can cover, beyond the custom fragments in registration messages and telemetry/command metadata, let's just not mention it for now and add it back when we can clearly define its purpose and scope.